.cirrus.yml

```diff
@@ -116,3 +158,19 @@ task:
     - name: tool_tests-macos
       env:
         SHARD: tool_tests
+
+
+docker_builder:
+  # Only build a new docker image when we tag a release (for dev, beta, or release.)
+  only_if: $CIRRUS_TAG != ''
```

Comment:
How is $CIRRUS_TAG set?

Reply:
From their docs: `CIRRUS_TAG: Tag name if current build was triggered by a new tag. For example v1.0`, so this should trigger whenever we tag dev, beta, or release builds.

Reply:
Hmm.  I probably should include the tag name in the Docker image name, I guess.

Reply:
I wonder if setup the Flutter in the Docker image would further accelerate the process. We can also update the image every day or every week to make the cache fresh so it will keep fast.

Reply:
Oh, wrong thread... I'm not familiar with the Github so my real question is: how do I trigger a build by a new tag?

Reply:
Just make a commit that adds a tag.  Just like we do when we roll dev or beta.

Reply:
And docker images can also be rebuilt by hand if it's been too long since the last dev roll (hopefully not necessary).